src/main/java/edu/kis/vh/nursery/IntArrayStack.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery;
+// kombinacja alt + <- lub alt + -> powoduje przełączanie się między zakładakmi (otwartymi klasami)
+public class IntArrayStack {
+	private static final int FULL = 11;
+
+	private static final int EMPTY_STACK_ERROR = -1;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)